setPoints: {
  first : [[0.0, 0.0, 0.0], [1.0, 0.0, 1.0], [2.0, 0.0, 1.0], [3.0, 0.0, 1.0]],
  straight_z : [[0.0, 0.0, 1.0], [0, 0, 2.0], [0, 0, 3.0], [0.0, 0.0, 4.0]],
}
eps : 0.3333333
v_max : 2
